data/genres.json

```diff
@@ -3,14 +3,14 @@
     "id": 448,
     "name": "Alternative",
     "items_ids": [
-      506
+
```

Comment:
Please put make sure this information is in this file, and stays there when the program loads and closes